Problem14.py

```diff
@@ -0,0 +1,26 @@
+class Solution:
+    def findMissingElement(self, nums):
+        low=0
+        high=len(nums)-1
+        if(len(nums)==1):
+            return "Nothing Missing"
+            
+        mid=(low+high)//2
+        if(nums[mid]-mid==1):
```

Comment:
There can missing elements on the right side

Reply:
I am checking that in else I think by moving the low to mid+1